models/Document/PageSnippet/Dao.php

```diff
@@ -54,9 +54,10 @@ public function getEditables()
             $editable->setDataFromResource($editableRaw['data']);
 
             $editables[$editableRaw['name']] = $editable;
-            $this->model->setEditable($editableRaw['name'], $editable);
         }
 
+        $this->model->setEditables($editables);
```

Comment:
I guess this can go as well ... since the caller uses the return value anyway: 
https://github.com/pimcore/pimcore/blob/338cd24e38090c8a13e0580228aa32b62f5bc860/models/Document/PageSnippet.php#L608

Reply:
In line 77/78 `$this->model->setVersions($versions);` should be removed, too, as it's the same.

Reply:
But actually... this pattern of setting the value on the from from inside the DAO is used in most of the DAO implementations... so maybe it should still be done - consistency wise - even if it's redundant...

Reply:
But then we should remove it from the other place, ... calling the same thing twice for nothing doesn't make sense 😉

Reply:
That's right. But maybe in a follow-up PR as it's not really related to this fix and might be a bigger changeset.

Reply:
you're right, this doesn't belong to this PR